src/components/standard/textFields/PETextFieldProps.ts

```diff
@@ -12,4 +12,9 @@ export interface PETextFieldProps {
     endContent?: ReactElement;
     style?: CSSProperties;
     className?: string;
+    inputProps?:
+        | {
+              [name: string]: number;
+          }
+        | undefined;
```

Comment:
The `?` in. line 15 already makes in ` | undefined`.
```suggestion
    inputProps?: {
          [name: string]: number;
    };
```